FirebaseCore.podspec

```diff
@@ -30,6 +30,7 @@ Firebase Core includes FIRApp and FIROptions which provide central configuration
   s.framework = 'Foundation'
   s.dependency 'GoogleUtilities/Environment', '~> 5.2'
   s.dependency 'GoogleUtilities/Logger', '~> 5.2'
+  s.dependency 'GoogleUtilities/Network', '~> 5.2'
```

Comment:
Unfortunately this is the only way to make `GULMutableDictionary` available in FirebaseCore. Probably, we should consider moving `GULMutableDictionary` to a separate pod (potentially together with similar classes).